view-creator-framework/build.gradle.kts

```diff
@@ -46,12 +46,15 @@ dependencies {
     implementation("org.glassfish.jersey.core:jersey-server:2.31") {
       because("XML Entity Expansion [High Severity][https://snyk.io/vuln/SNYK-JAVA-ORGGLASSFISHJERSEYMEDIA-595972] in org.glassfish.jersey.media:jersey-media-jaxb@2.28")
     }
-    implementation("io.grpc:grpc-core:1.31.0") {
+    implementation("io.grpc:grpc-core:1.33.1") {
       because("Information Exposure [Medium Severity][https://snyk.io/vuln/SNYK-JAVA-IOGRPC-571957] in io.grpc:grpc-core@1.30.0")
     }
     implementation("com.google.guava:guava:30.0-android") {
       because("Information Disclosure (new) [Medium Severity][https://snyk.io/vuln/SNYK-JAVA-COMGOOGLEGUAVA-1015415] in com.google.guava:guava@29.0-android")
     }
+    implementation("org.apache.calcite:calcite-core:1.26.0") {
```

Comment:
instead, constrain the highest order calcite dependency (I believe it's calcite-babel from another repo). The idea there is if we're going to upgrade part of calcite, we might as well upgrade the whole suite of artifacts since they may have versioned codependencies.

Reply:
Not sure I get you, so you want to change this to calcite-babel? Won't that be confusing since the vulnerability is in calcite-core?

Reply:
So imagine a scenario (similar to what we have) where there's a vulnerability in netty, but we're using grpc netty, there'd we'd probably want to upgrade all grpc libs, because we don't want to just upgrade grpc netty without upgrading the other grpc libs. This is similar, although a level down, since calcite itself is a transitive dependency.

Reply:
ok.